druid/src/env.rs

```diff
@@ -208,6 +208,39 @@ impl Env {
             .map(|value| value.to_inner_unchecked())
     }
 
+    /// Gets a value from the environment, in its encapsulated [`Value`] form,
+    /// expecting the key to be present.
+    ///
+    /// *WARNING:* This is not intended for general use, but only for inspecting an `Env` e.g.
+    /// for debugging, theme editing, and theme loading.
+    /// [`Value`]: enum.Value.html
+    pub fn get_untyped(&self, key: impl Borrow<Key<()>>) -> &Value {
+        let key = key.borrow();
+        if let Some(value) = self.0.map.get(key.key) {
+            value
+        } else {
+            panic!("key for {} not found", key.key)
```

Comment:
Let's document panic conditions under the *Panics* header. Take a look at some of the other functions here like `try_get`.